### ISSUE TITLE
fMLLR Alignements

### ISSUE THREAD

**Issue by mohitsharma29** [mravanelli/pytorch-kaldi#89]
I am training a speaker, independent model. Is there any reason as to why all tutorials using fMLLR alignments, which is speaker adaptive?

**Comment by mravanelli** [comment]
alignments are only the labels used by the neural network. You can use all
the alignments you want actually, but it is important to have rather good
alignments for training a good model. Note that you can train a speaker
independent model also with alignments computed on fmllr model.
